lib/utils/HTTP/login_requests.dart

```
@@ -0,0 +1,73 @@
+// Dart imports:
+import 'dart:convert';
+
+// Flutter imports:
+import 'package:flutter/foundation.dart';
+
+// Package imports:
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:http/http.dart' as http;
+
+// Project imports:
+import 'package:exdock_backend_client/utils/HTTP/http_data.dart';
+import 'package:exdock_backend_client/utils/authentication/authentication_data.dart';
+
+Future<HttpData> loginRequest(AuthenticationData authData) async {
+  FlutterSecureStorage storage = const FlutterSecureStorage();
+  int statusCode = 404;
```

throw instead of returning 404 on error